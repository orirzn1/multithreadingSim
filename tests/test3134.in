L11
S6
O14
N13

T0
I@0x00000000
STORE $5, $2, 0x3
LOAD $4, $7, $7
SUB $6, $5, $0
LOAD $7, $0, $6
SUB $4, $3, $0
HALT $0

T1
I@0x00000000
ADDI $1, $7, 0x3
ADD $1, $6, $4
LOAD $7, $6, 0x4
SUBI $5, $7, 0x1
ADDI $1, $7, 0x4
ADDI $7, $1, 0x4
HALT $0

T2
I@0x00000000
STORE $5, $6, 0x4
STORE $5, $6, 0x2
ADD $0, $0, $5
ADDI $1, $7, 0x2
ADDI $1, $1, 0x2
SUBI $7, $5, 0x1
SUB $4, $4, $4
SUB $1, $4, $6
HALT $0

T3
I@0x00000000
STORE $1, $6, $1
SUB $3, $4, $4
STORE $3, $0, 0x4
STORE $1, $5, 0x1
ADDI $2, $3, 0x4
SUBI $7, $5, 0x1
LOAD $5, $2, $1
HALT $0

T4
I@0x00000000
LOAD $4, $7, 0x2
SUB $0, $3, $6
STORE $0, $6, 0x1
SUBI $0, $4, 0x1
SUB $2, $0, $4
ADD $4, $5, $5
ADD $5, $2, $2
HALT $0

T5
I@0x00000000
ADDI $3, $3, 0x4
LOAD $6, $4, $3
ADD $3, $0, $1
STORE $5, $6, $5
SUBI $1, $6, 0x2
HALT $0

T6
I@0x00000000
SUB $0, $0, $1
LOAD $2, $1, 0x1
SUB $5, $6, $2
ADD $7, $3, $0
HALT $0

T7
I@0x00000000
LOAD $2, $0, 0x3
SUBI $2, $5, 0x2
SUBI $2, $3, 0x4
ADD $0, $3, $3
STORE $5, $2, 0x1
ADD $7, $6, $6
ADD $6, $6, $4
SUBI $1, $3, 0x3
SUBI $6, $5, 0x1
ADD $3, $4, $6
HALT $0

T8
I@0x00000000
SUBI $7, $5, 0x4
ADD $3, $1, $7
SUB $0, $0, $3
LOAD $1, $5, 0x3
SUBI $5, $7, 0x1
HALT $0

T9
I@0x00000000
STORE $5, $0, 0x1
SUB $6, $0, $1
STORE $4, $7, $1
STORE $1, $7, 0x0
LOAD $3, $2, 0x2
HALT $0

T10
I@0x00000000
ADDI $7, $7, 0x4
STORE $6, $0, $2
STORE $4, $7, $2
STORE $2, $1, 0x0
STORE $0, $3, $7
ADD $0, $0, $1
ADD $1, $5, $2
HALT $0

T11
I@0x00000000
SUBI $0, $7, 0x3
ADD $4, $6, $5
STORE $4, $7, 0x4
ADD $4, $5, $0
ADDI $2, $5, 0x1
ADD $6, $3, $2
ADD $7, $1, $7
ADDI $7, $1, 0x1
SUBI $4, $0, 0x4
HALT $0

T12
I@0x00000000
ADDI $1, $5, 0x2
ADD $7, $4, $6
ADD $5, $5, $1
STORE $2, $6, 0x0
SUB $5, $7, $5
ADD $1, $0, $4
STORE $6, $3, 0x0
SUBI $2, $5, 0x4
STORE $0, $3, $5
STORE $1, $7, $6
HALT $0

D@0x0
0x3
0x1
0x2
0x1
0x2
