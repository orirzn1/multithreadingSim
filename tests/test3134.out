
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x5	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x9
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFF
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFF
Register file thread id 4:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFC	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 8:
	R0 = 0x4	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0x0	R5 = 0xFFFFFFFB	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 10:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x4
Register file thread id 11:
	R0 = 0xFFFFFFFD	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0xFFFFFFF9	R5 = 0x0	R6 = 0x1	R7 = 0x1
Register file thread id 12:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFA	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 6.960396

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x4
Register file thread id 1:
	R0 = 0x0	R1 = 0x4	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFF	R6 = 0x0	R7 = 0x8
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFF
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x4	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0xFFFFFFFF
Register file thread id 4:
	R0 = 0xFFFFFFFF	R1 = 0x0	R2 = 0xFFFFFFFF	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0xFFFFFFFE	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 6:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 7:
	R0 = 0x0	R1 = 0xFFFFFFFD	R2 = 0xFFFFFFFC	R3 = 0xFFFFFFFF	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 8:
	R0 = 0x4	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0x0	R5 = 0xFFFFFFFB	R6 = 0x0	R7 = 0xFFFFFFFC
Register file thread id 9:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 10:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x4
Register file thread id 11:
	R0 = 0xFFFFFFFD	R1 = 0x0	R2 = 0x1	R3 = 0x0	R4 = 0xFFFFFFF9	R5 = 0x0	R6 = 0x1	R7 = 0x1
Register file thread id 12:
	R0 = 0x0	R1 = 0x0	R2 = 0xFFFFFFFA	R3 = 0x0	R4 = 0x0	R5 = 0xFFFFFFFE	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.108911

